tests/sky_level_reference_values.py

```diff
@@ -19,12 +22,14 @@
 sky_model = skybrightness.SkyModel()
 with warnings.catch_warnings():
     warnings.simplefilter('ignore')
-    sky_model.setRaDecMjd(ra, dec, mjd, degrees=True)
+    sky_model.set_ra_dec_mjd(ra, dec, mjd, degrees=True)
 
 sky_levels = {}
 for band in 'ugrizy':
-    bandpass = galsim.RubinBandpass(band)
-    wave, spec = sky_model.returnWaveSpec()
+    with mock.patch('os.getenv', return_value=''):
```

Comment:
I think there is probably no need to do this.  Should probably just use the regular RubinBandpass, rather than the old GalSim ones.

Reply:
Then also remove the mocking in test_sky_model.

Reply:
Then we need to change the code in test_sky_model also.

Reply:
Right.  Since we learned we can't actually do the regression test we thought we were doing, I don't see any point in using the galsim Bandpass for that test.

Reply:
BTW, I added a test to test_bandpass.py that checks the fallback to GalSim's files when RUBIN_SIM_DATA is not defined.